tools/train.py

```diff
@@ -183,7 +193,11 @@ def main():
         distributed = False
     else:
         distributed = True
-        init_dist(args.launcher, **cfg.dist_params)
+        if torch.cuda.is_available():
+            init_dist(args.launcher, **cfg.dist_params)
+        else:
+            cfg.dist_params['backend'] = 'gloo'
```

Comment:
Does this introduce an extra dependency or anything that's worth mentioning in the docs?

Reply:
By default for Linux, the Gloo and NCCL backends are built and included in PyTorch distributed (NCCL only when building with CUDA).